src/tritonserver.cc

```diff
@@ -2542,6 +2542,18 @@ TRITONSERVER_ServerStop(TRITONSERVER_Server* server)
   return nullptr;  // Success
 }
 
+TRITONAPI_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_ServerStopWithTimeout(
+    TRITONSERVER_Server* server, unsigned int timeout)
+{
+  tc::InferenceServer* lserver = reinterpret_cast<tc::InferenceServer*>(server);
+  if (lserver != nullptr) {
+    lserver->SetExitTimeoutSeconds(timeout);
```

Comment:
@kthui - I didn't realize that the internal object had a method `SetExitTimeoutSeconds` - this being the case I think also we could expose this method directly as well - that is add a new API
```
TRITONSERVER_ServerSetExitTimeoutSeconds()
```
I think in the design doc I thought we had something like "update" and that seemed odd - sorry for the thrash on this.

Reply:
Sure, updated to expose the `SetExitTimeoutSeconds` method instead. The API name is
```
TRITONSERVER_ServerSetExitTimeout()
```
to align with the API name that sets the opaque option object when creating the server object
```
TRITONSERVER_ServerOptionsSetExitTimeout()
```